.github/workflows/label-checker.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Label Checker
+on:
+  pull_request:
```

Comment:
I think you need `pull_request_target` here so that it applies to PR's coming from forks. Right?

Reply:
`pull_request_target` is for specifying directories, branches, etc. So if we had a `release` branch and only wanted this to run when that was the merge target, we'd need `pull_request_target`. I think this will still work on fork'd PRs, but we can always check that and adjust once it's live. It's one of those things that'll be pretty obvious once we have it up. I only did tests from my own fork into my own fork.